client/src/components/_common/Message/Message.js

```diff
@@ -74,7 +75,7 @@ const Message = ({
   const typeMap = TYPE_MAP[type];
   const scopeMap = SCOPE_MAP[scope || DEFAULT_SCOPE];
   const { iconName, iconText, className: typeClassName } = typeMap;
-  const { role, tagName, className: scopeClassName } = scopeMap;
+  const { role, tagName: defaultTagName, className: scopeClassName } = scopeMap;
```

Comment:
What's the use case for setting a defaultTagName as opposed to leaving it blank and then explicitly setting it as 'small'?

Reply:
_**Clarification**: In the code, the `tagName` from `scopeMap` is being renamed to `defaultTagName`._

The use case for a standard `tagName` (per scope) (as defined in [`SCOPE_MAP`](https://github.com/TACC/Frontera-Portal/blob/51ea063414d7b588f42ba7a29617d3eab9eea668/client/src/components/_common/Message/Message.js#L42-L52)) is that:
- an "inline message" should use a semantically accurate tag whose default `display` is `inline` (`<span>`)
- a "section message" should use a semantically accurate tag whose default `display` is `block` (`<p>`)

I do not know how one can be "leaving it blank" while also "explicitly setting".

The `tagName` has only just been renamed to `defaultTagName` in this context, in this PR, to distinguish it from the `tagName` prop. And, now that I read this code again, I prefer to use the variable name `standardTagName`, instead.

Reply:
Oh.

- The component user leaves the `tagName` empty (does not set `tagName`).
- The component author sets the default value for `tagName` to `small`.

Is that an accurate understanding of your inquiry? If so, then the problems that arise are:

- The component user would acquire `<small>` through undocumented means.
- The component author would lack a rule to determine when to use standard tagName over `<small>`, but the component author does not want to support `<small>` to begin with.